src/decorators/legacy/model-hooks-single.ts

```diff
@@ -0,0 +1,134 @@
+import type { HookOptions, HookDecoratorArgs } from './hook-decorators.js';
```

Comment:
Looking at https://github.com/sequelize/sequelize/blob/a4f8e62dd5b5b1c9df983b201581165d8465f107/src/model-typescript.ts#L102-L118 I'm still missing the `Associate` hooks

Reply:
Done